pages/index.tsx

```diff
@@ -104,8 +113,7 @@ const Home: NextPage<Props> = ({ posts, query, tinaData, variables }) => {
       />
       <section className="simple-section">
         <div className="container">
-          {/* TODO: Implement tags count for the admin user */}
-          <Tags uniqueTags={uniqueSortedTags} />
+          <Tags sortedTags={sortedTags} />
```

Comment:
Maybe we don't need `sorted` prefix in this variable name

Those are just tags